languages/my/init.lua

```diff
@@ -1,3 +1,10 @@
+local unicode = require("languages.unicode")
+
+local language = pl.class(unicode)
+language._name = "my"
+
+-- TODO tokenizer never used or tested!
```

Comment:
Indeed, not called anywhere in the previous code base either.
--> Do we have an open issue on the topic?

Reply:
#2255